flask_rebar/rebar.py

```diff
@@ -16,6 +16,8 @@
 from copy import copy
 from distutils.version import LooseVersion
 from functools import wraps
+from inspect import isclass
+from types import MethodType
```

Comment:
```suggestion
```
unused import

Reply:
Whoops guess we should maybe add Flake8 to the pre-commit hooks :)